editor/header/publish-button/style.scss

```diff
@@ -0,0 +1,35 @@
+// Needs specificity to override bleed
+.wp-core-ui .button.button-large.editor-publish-button {
+	margin: 0 10px;
+}
+
+.editor-publish-button.is-saving,
+.editor-publish-button.is-saving:disabled {
+	position: relative;
+
+	// These styles overrides the disabled state with the button primary styles
```

Comment:
@jasmussen is this something we need upstream?

Reply:
Good question. I don't know what the best approach is, here — looking at it now I'm worried that we might have a blue disabled button, even if a user has a different admin color scheme.

Long term one would hope we had a single "button-primary" component that has this all baked in, instead of multiple disparate CSS that's competing.